### ISSUE TITLE
🗄️ [Documentation] - Point PikaPods users to how to install the PWA.

### ISSUE THREAD

**Issue by youngcw** [actualbudget/docs#466]
### Description

For those that use PikaPods, the PWA is the best options for a desktop app.  It probably would be good to add that to the docs in some way.

### Category

Installation & Configuration

**Comment by RubenOlsen** [comment]
@youngcw is this still relevant given that we now have proper desktop packages?

**Comment by youngcw** [comment]
yes.  Its still best that if you host a server somewhere you use the PWA instead of the desktop app to not have version issues.

**Comment by RubenOlsen** [comment]
Ok then. I'll think of something that caters both to highly technical users and others.
